src/linkedlist.ts

```diff
@@ -1,5 +1,15 @@
 export class Cons<A> {
```

Comment:
Can we think of a better name for this file now that it is no longer just a linked list? `datastructures.ts`? `lists.ts`?

Reply:
I agree that it should be renamed. I would prefer `datastructures.ts`